charts/country-risk/values.yaml

```diff
@@ -0,0 +1,367 @@
+  nodeSelector: {}
+
+  # -- Pod toleration constraints
+  tolerations: []
+
+  # Following Catena-X Helm Best Practices @url: https://catenax-ng.github.io/docs/kubernetes-basics/helm
+  # @url: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity
+  affinity:
+    podAntiAffinity:
+      preferredDuringSchedulingIgnoredDuringExecution:
+        - weight: 100
+          podAffinityTerm:
+            labelSelector:
+              matchExpressions:
+                  # -- Match Pod rules
+                - key: app.kubernetes.io/name
+                  operator: DoesNotExist
+            # -- Key that is used to determine the topology of the cluster
+            topologyKey: kubernetes.io/hostname
+
+  # Following Catena-X Helm Best Practices @url: https://catenax-ng.github.io/docs/kubernetes-basics/helm
+  # @url: https://github.com/helm/charts/blob/master/stable/nginx-ingress/values.yaml#L210
+  # -- Determines if a pod is still alive or not
+  livenessProbe:
+    failureThreshold: 5
+    # -- Number of seconds to wait before performing the first liveness probe
+    initialDelaySeconds: 120
+    # -- Number of seconds to wait between consecutive probes
+    periodSeconds: 10
+    # -- Number of consecutive successful probes before a pod is considered healthy
+    successThreshold: 1
+    # -- Number of seconds after which a liveness probe times out
+    timeoutSeconds: 1
+    # -- HTTP endpoint
+    path: /management/health/liveness
+    # -- Port used
+    port: 8080
+  # -- Determine when a pod is ready to start accepting requests
+  readinessProbe:
+    failureThreshold: 3
+    initialDelaySeconds: 60
+    periodSeconds: 10
+    successThreshold: 1
+    timeoutSeconds: 1
+    path: /management/health/readiness
+    port: 8080
+
+  # -- Defines the client secret and client ID
+  applicationSecret:
+    # -- Value that specifies whether the application secret should be used
+    enabled: true
+    # -- String value that represents the client secret
+    clientSecret: ""  #
+    # -- String value that represents the client ID
+    clientId: ""  #
+
+  configmap:
+    # Specifies whether a configmap should be created
+    create: true
+    data:
+      # -- Which profile should be activated for the application
+      spring_profiles_active: 'dev'
+      # -- Security configurations for the application
+      security_enabled: 'false'
+
+
+  elastic:
+    # -- Should elastic be enabled or not
+    enabled: false
+    security:
+      # -- Information about the transport layer security (TLS)
+      tls:
+        # -- Encryption for the REST requests made to the Elastic cluster
+        restEncryption: false
+
+
+#########################
+# Postgres configuration     #
+#########################
+
+postgres:
+  enabled: true
+  auth:
+    database: vas
+    # overwrite for security reasons
+    postgresPassword: default
+    username: vas
+    # overwrite for security reasons
+    password: default
```

Comment:
this was just a simply default password so when no profile is passed it can connect the microservice to the postgres, in other value envs it will get them from vault